model/textparse/promparse_test.go

```diff
@@ -498,8 +498,16 @@ func BenchmarkParse(b *testing.B) {
 		"openmetrics": func(b []byte, st *labels.SymbolTable) Parser {
 			return NewOpenMetricsParser(b, st)
 		},
+		"openmetrics/skip-ct": func(b []byte, st *labels.SymbolTable) Parser {
+			return NewOpenMetricsParser(b, st, WithOMParserCTSeriesSkipped())
+		},
 	} {
-		for _, fn := range []string{"promtestdata.txt", "promtestdata.nometa.txt"} {
+		for _, fn := range []string{"promtestdata.txt", "promtestdata.nometa.txt", "omtestdata.txt"} {
+			// we only want to benchmark om parsers with omtestdata
+			if fn == "omtestdata.txt" && parserName == "prometheus" {
```

Comment:
```suggestion
			if fn == "omtestdata.txt" && !strings.HasPrefix(fn, "openmetrics") {
```

I think we could make this more future-proof in case we add more things here 🤔

Reply:
yes this is better!